xrpl/asyncio/transaction/main.py

```diff
@@ -269,18 +269,13 @@ async def _check_fee(transaction: Transaction, client: Optional[Client] = None)
     Raises:
         XRPLException: if the transaction fee is higher than the expected fee.
     """
-    # Calculate the expected fee from the network load and transaction type
-    expected_fee = await _calculate_fee_per_transaction_type(transaction, client)
```

Comment:
Should `_calculate_fee_per_transaction_type` be removed?

Reply:
No, it's used for the autofill fee calculation